### PR TITLE
Fix NATS contrib.eventing.knative.dev/release labels

### LINKED ISSUE #527: Fix NATS eventing.knative.dev/release labels
## Expected Behavior
NATS labels to be `contrib.eventing.knative.dev/release`

## Actual Behavior
`eventing.knative.dev/release`

### PR DESCRIPTION
**Pull request by montaro**
Fixes #527 

```release-note
Fix NATS contrib.eventing.knative.dev/release labels
```

### CODE REVIEW

**Comment by montaro** [comment]
@grantr Please check

File: natss/config/300-natss-channel.yaml [hunk 1]
```diff
@@ -17,9 +17,9 @@ kind: CustomResourceDefinition
 metadata:
  name: natsschannels.messaging.knative.dev
```
Comment (n3wscott) [inline natss/config/300-natss-channel.yaml]: the spacing here is off

Comment (montaro) [reply natss/config/300-natss-channel.yaml]: fixed

**Pull request merged**